fullstack-templates/saas/backend/src/auth.rs

```diff
@@ -74,7 +74,7 @@ pub async fn login(
                 .expect("Couldn't insert session :(");
 
             let cookie = Cookie::build(("foo", session_id))
-                .secure(true)
+                .secure(!cfg!(debug_assertions))
```

Comment:
Can you add a small comment above this line to explain the reason for it?

Reply:
I have added the comment "// Only send the cookie over HTTPS in production", the problem here was that no cookies were stored locally due to the missing https